Mathlib/Data/Vector/Basic.lean

```diff
@@ -696,24 +696,6 @@ instance : LawfulTraversable.{u} (flip Vector n) where
   comp_map := by intro _ _ _ _ _ x; cases x; simp! [(· <$> ·)]
   map_const := rfl
 
--- Porting note: not porting meta instances
```

Comment:
`reflect` seems to have no equivalent in Lean 4, so I propose deleting this outright.